src/components/Navbar/Hamburger.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Link from 'next/link'
+import NavbarAccount from './Account'
+import { useRouter } from 'next/router'
+
+const Hamburger = () => {
+  const router = useRouter()
+
+  const handleProductClick = () => {
+    router.push('/products')
+  }
+
+  const handleAboutUsClick = () => {
+    router.push('/about-us')
+  }
+
+  const handleSignInClick = () => {
+    router.push('/sign-in')
```

Comment:
We are not having `Sign-in` screen but the modal instead, can you add `TODO: Open sign-in modal` here for others to understand